index.md

```diff
@@ -5,49 +5,20 @@ title: OWASP Code Review Guide
 tags: Code Review Guide
 level: 3
 type: documentation
-auto-migrated: 1
 
 ---
 
-The current PDF version can be found [here](/www-pdf-archive/OWASP_Code_Review_Guide_v2.pdf)
+[![OWASP Flagship](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)

```

Comment:
Should be OWASP Lab no?

Reply:
Yup

Reply:
Oh you mean the link, Doesn't seem like linking to the lab heading works.

Reply:
```suggestion
[![OWASP Lab](https://img.shields.io/badge/owasp-lab-yellow.svg)](https://owasp.org/projects/#all-projects)

```

Reply:
Oh Tks!